### PR TITLE
docs: update to ccs-only addon testing

### PR DESCRIPTION
**Pull request by whereswaldon**
Do not merge this until we get confirmation from leadership that all addon testing should be CCS-based.

/hold

### CODE REVIEW

File: docs/Addons.md [hunk 1]
```diff
@@ -18,86 +18,33 @@ The [Prow Operator Test] is a good example of a [Basic operator test]. It verifi
...
-
-```yaml
----
-$schema: /user-1.yaml
-
-user_id: "your-ocm-user"
+### Providing Secrets to Your Build
 
-kerberos_id: "N/A"
+Follow the documentation [here](https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/) to create secrets and configure them to be mirrored into the `ci` namespace [like ours](https://github.com/openshift/release/blob/master/core-services/secret-mirroring/_mapping.yaml#L62).
```
Comment (k-wall) [inline docs/Addons.md:39]: Following the link, I read "All users who wish to interact with the system must log in to the api.ci cluster", following the console link and chosing "git hub" as the identity provider, I see "An authentication error occurred.".   Could the doc make it clear how an addon author would go about getting permission?

Comment (k-wall) [inline docs/Addons.md:39]: Is there a naming convention for this secret?

Comment (whereswaldon) [reply docs/Addons.md:39]: Ehh, not really. Most people use `<addon-name>-credentials`

Comment (whereswaldon) [reply docs/Addons.md:41]: I've pushed a change adding instructions for joining the OpenShift organization, which I believe is required for this.

**Comment by whereswaldon** [comment]
This is the direction we're going.
/hold cancel

**Pull request merged**